### PR TITLE
Update Auditing and Data Classification SDK clients

### PR DESCRIPTION
**Pull request by bashahee**
Updating SDK clients of Auditing and Data Classification services.

### CODE REVIEW

File: sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Sql</PackageId>
 		<Description>Azure SQL Management SDK library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-		<Version>1.40.0-preview</Version>
+		<Version>1.41.0-preview</Version>
 		<PackageTags>Microsoft Azure SQL Management;SQL;SQL Management;</PackageTags>
 		<PackageReleaseNotes>
 			<![CDATA[
```
Comment (isra-fel) [inline sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj]: Please edit the release notes

File: sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/SDKs/_metadata/sql_resource-manager.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿Installing AutoRest version: 2.0.4413
```
Comment (isra-fel) [inline sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/SDKs/_metadata/sql_resource-manager.txt]: This file should be in `/eng/mgmt/mgmtmetadata`

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @bashahee , please write tests to cover the new generated operations. Thanks

**Comment by bashahee** [comment]
@isra-fel No new operations were generated, few models were extended.
Hopefully, I managed to address your comments.

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**